utilities/create-mock-data/ModelFactories/BaseDocumentUploadFactory.cs

```diff
@@ -0,0 +1,32 @@
+using Bogus;
+using Prime.Models;
+
+namespace PrimeTests.ModelFactories
+{
+    public class BusinessLicenceDocumentFactory : BaseDocumentUploadFactory<BusinessLicenceDocument>
+    {
+        public BusinessLicenceDocumentFactory(BusinessLicence related)
+        {
+//            this.SetBaseRules();
+
+            RuleFor(x => x.BusinessLicence, f => related);
+
+            Ignore(x => x.BusinessLicenceId);
+        }
+    }
+
+
+    public abstract class BaseDocumentUploadFactory<T> : Faker<T> where T : BaseDocumentUpload
+    {
+        protected BaseDocumentUploadFactory()
+        {
+//            this.SetBaseRules();
```

Comment:
Commenting out rather than removing the code is just to clearly show the handful of changes that need to be made (documented in `README.md`) in order to re-use the factories from `prime-dotnet-webapi-tests`